docs/cosmology/index.rst

```diff
@@ -166,7 +167,7 @@ They also accept arrays of redshifts:
 
 .. doctest-requires:: scipy
 
-  >>> cosmo.age([0.5, 1, 1.5]).value  # doctest: +FLOAT_CMP
+  >>> cosmo.age(np.array([0.5, 1, 1.5])).value  # doctest: +FLOAT_CMP
```

Comment:
I wonder if you still want to use `np.array` here - I guess it is best practice, but perhaps then encourage using `Quantity` and write as `[0.5, 1, 1.5] * cu.redshift`?

Separately, I know it was already here, but why the `.value` here? Would seem better to show the `Quantity` output.

Reply:
SGTM.
Pushed.

Reply:
Passed.
Merging.